src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java

```diff
@@ -204,9 +205,17 @@ private BeachfrontBannerRequest getBannerRequest(BidRequest bidRequest, List<Imp
             requestBuilder.secure(firstImpSecure != null ? firstImpSecure : getSecure(bundle));
         }
 
+        if (bidRequest.getSource() != null && bidRequest.getSource().getExt() != null) {
+            requestBuilder.schain(getSchain(bidRequest));
+        }
+
         return requestBuilder.build();
     }
 
+    private ExtRequestPrebidSchainSchain getSchain(BidRequest bidRequest) {
+        return bidRequest.getSource().getExt().getSchain();
+    }
```

Comment:
Actually we can make this if statement much more shorter, just to first extract schain and then check it for null
```java
        if (schain != null) {
            requestBuilder.schain(schain);
        }
      ...
    }

    private static ExtRequestPrebidSchainSchain getSchain(BidRequest bidRequest) {
        final Source source = bidRequest.getSource();
        final ExtSource extSource = source != null ? source.getExt() : null;

        return extSource != null ? extSource.getSchain() : null;
    }
```